sound/soc/sof/ipc4.c

```diff
@@ -665,5 +668,5 @@ const struct sof_ipc_ops ipc4_ops = {
 	.fw_loader = &ipc4_loader_ops,
 	.tplg = &ipc4_tplg_ops,
 	.pcm = &ipc4_pcm_ops,
-	.fw_tracing = &ipc4_mtrace_ops,
+	.fw_tracing = &ipc3_dtrace_ops,
```

Comment:
yuk. that's horrible.

Reply:
@plbossart yup, fully agree, this is not a solution for merging.